build/ol/install-packages.sh

```diff
@@ -4,56 +4,46 @@ export TARGET
 
 set -euo pipefail
 
+# Install and configure dnf
 microdnf install dnf -y
-
 dnf install 'dnf-command(config-manager)' -y
 dnf config-manager --set-enabled ol8_codeready_builder
-tee /etc/yum.repos.d/ol8-epel.repo<<EOF
+
+# Add EPEL repository
+tee /etc/yum.repos.d/ol8-epel.repo <<EOF
 [ol8_developer_EPEL]
-name= Oracle Linux \$releasever EPEL (\$basearch)
+name=Oracle Linux \$releasever EPEL (\$basearch)
 baseurl=https://yum.oracle.com/repo/OracleLinux/OL8/developer/EPEL/\$basearch/
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-oracle
 gpgcheck=1
 enabled=1
 EOF
-dnf update -y
-
-dnf install -y \
-  ImageMagick \
-  file \
-  sudo \
-  net-tools \
-  iputils \
-  curl \
-  git \
-  git-lfs \
-  jq \
-  dos2unix \
-  mysql \
-  procps-ng \
-  tzdata \
-  rsync \
-  nano \
-  unzip \
-  zstd \
-  lbzip2 \
-  libpcap \
-  libwebp \
-  findutils \
-  which
 
+# Update and install necessary packages
+dnf update -y && dnf install -y \
+  ImageMagick file sudo net-tools iputils curl git jq dos2unix mysql \
+  procps-ng tzdata rsync nano unzip zstd lbzip2 libpcap libwebp \
+  findutils which
+
+# Install Git LFS
 curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.rpm.sh | sudo bash
-dnf update -y
```

Comment:
This line got lost

Reply:
Sorry about this. Not sure how I did that.